### PR TITLE
helper-cli: Add a command to export curations from package configs

### PR DESCRIPTION
**Pull request by fviernau**
This enables a workflow similar to what we already have for repository configurations.

### CODE REVIEW

**Review by MarcelBochtler — APPROVED** [review]
LGTM

**Comment by sschuberth** [comment]
Please note that this conflicts with https://github.com/oss-review-toolkit/ort/pull/3874, but I can rebase my PR (yet again).

File: helper-cli/src/main/kotlin/commands/packageconfig/ExportLicenseFindingCurationsCommand.kt [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    override fun run() {
+        val localLicenseFindingCurations = getLicenseFindingCurationsByRepository(
+            curations = packageConfigurationFile.readValue<PackageConfiguration>().licenseFindingCurations,
+            nestedRepositories = findRepositories(sourceCodeDir)
+        )
+
+        val globalLicenseFindingCurations = if (licenseFindingCurationsFile.isFile) {
+            licenseFindingCurationsFile.readValue<RepositoryLicenseFindingCurations>()
+        } else {
+            mapOf()
```
Comment (mnonnenmacher) [inline helper-cli/src/main/kotlin/commands/packageconfig/ExportLicenseFindingCurationsCommand.kt:81]: Better: `emptyMap()`

**Review by mnonnenmacher — CHANGES_REQUESTED** [review]

File: helper-cli/src/main/kotlin/commands/packageconfig/ExportLicenseFindingCurationsCommand.kt (hunk 1 above)
Comment (sschuberth) [reply helper-cli/src/main/kotlin/commands/packageconfig/ExportLicenseFindingCurationsCommand.kt:81]: Or yet better (IMO), follow the approach outlined here:

https://github.com/oss-review-toolkit/ort/pull/3874/files#diff-fbfa4ac54e65cb37598fe5875b4c25eaa90f68955a06c9038f663599e983aad2R82-R83

**Review by mnonnenmacher — APPROVED** [review]

File: helper-cli/src/main/kotlin/commands/ExportLicenseFindingCurationsCommand.kt [hunk 2]
```diff
@@ -75,11 +75,8 @@ internal class ExportLicenseFindingCurationsCommand : CliktCommand(
             nestedRepositories = ortResult.repository.nestedRepositories
         )
 
-        val globalLicenseFindingCurations = if (licenseFindingCurationsFile.isFile) {
-            licenseFindingCurationsFile.readValue<RepositoryLicenseFindingCurations>()
-        } else {
-            mapOf()
-        }
+        val globalLicenseFindingCurations = licenseFindingCurationsFile.takeIf { it.isFile }
+            ?.readValue<RepositoryLicenseFindingCurations>().orEmpty()
```
Comment (sschuberth) [inline helper-cli/src/main/kotlin/commands/ExportLicenseFindingCurationsCommand.kt:79]: Shouldn't this have gone to `helper-cli/src/main/kotlin/commands/packageconfig/ExportLicenseFindingCurationsCommand.kt` instead?

**Review by sschuberth — CHANGES_REQUESTED** [review]

File: helper-cli/src/main/kotlin/commands/ExportLicenseFindingCurationsCommand.kt (hunk 2 above)
Comment (fviernau) [reply helper-cli/src/main/kotlin/commands/ExportLicenseFindingCurationsCommand.kt:79]: Argh right, good catch!

Comment (sschuberth) [reply helper-cli/src/main/kotlin/commands/ExportLicenseFindingCurationsCommand.kt:79]: Ok, you now also changed the other location, but this change it still there...

**Review by sschuberth — APPROVED** [review]

**Pull request merged**